### DISCUSSION TITLE
get url traffic via API

### DISCUSSION THREAD

**Discussion by jonahktjala** [YOURLS/YOURLS#3223] [Q&A] [answered]
Hi, I am interested on creating some API calls to get traffic performance of my shortlinks. I have been checking on the list of plugins but it seems like it's not there yet. Use case is I wanted to do some API Calls to help ease my regular count of traffic on 20+ URLS. 
Could someone help point me to the right direction? Thank you lots :) 
Love YOURLS btw <3

**Comment by dgw** [comment, marked as answer]
Near-duplicate of #2879 and/or #3113.

See [API Usage section](https://yourls.org/#API;apiusage) of the website. Sounds like you want the `url-stats` action.

**Reply by jonahktjala** [reply]
Great! Thank you very much :)
